src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -69,12 +69,20 @@ Not every language or logging framework is supported yet. The following are our
 * [Python logs in context procedures](/docs/logs/logs-context/configure-logs-context-python) for agent [v7.12.0.176](/docs/release-notes/agent-release-notes/python-release-notes)
 * [Ruby logs in context procedures](/docs/logs/logs-context/configure-logs-context-ruby) for agent [v8.6.0 or higher](/docs/release-notes/agent-release-notes/ruby-release-notes)
 
-<Callout variant="tip">
-  APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
-</Callout>
-
 If your APM agent doesn't support our automatic logs in context solution yet, you can continue to use our [manual logs in context solutions](/docs/logs/logs-context/logs-in-context), and forward your logs via our infrastructure agent or supported third-party forwarder.
 
+## Known limitations [#limitations]
+
+APM logs in context automatically forwards APM agent log data and is enabled by default. This can have a negative impact on your security, compliance, billing, or system performance. For more information, or if you need to adjust the default setting, follow the procedures to [disable automatic logging](/docs/logs/logs-context/disable-automatic-logging).
+
+Here are some additional known limitations:
+
+* Startup logs or host information are not available. This is because logs are not collected until after the app is loaded.
+* If you are using Kubernetes, the logs will not get decoration from the Kubernetes API. This is because the logs never touch the host or exist in a place where the API can be called.
+* When an APM event is triggered, you may not see all stack traces in the associated logs in context for Java or .NET agents. As a workaround, you can change your drop filter rules.
+
+For example, Fluentd can add the `processID` from the entity that generated the log, but APM logs can't see that. In Kubernetes, the API is called to add metadata, but that can't be seen from within the application. In these situations, we recommend that you use automatic logs in context but do not ship the logs from the application. Instead, continue to use Fluentd, Fluent Bit, or another solution to forward the log files.
```

Comment:
The location/indentation on this paragraph confused me a bit; is this example specific to the last bullet mentioned or something else? It wasn't clear to me what it was related to.

Reply:
@kford-newrelic  Good point. I'll double check.